UIKitDz/ViewController.swift

```diff
@@ -5,8 +5,72 @@ import UIKit
 
 /// Class Controller
 class ViewController: UIViewController {
```

Comment:
final - ставим перед всеми классами для производительности и ускорения горячего билда.
```suggestion
class ViewController: UIViewController {
```